repos/system_upgrade/common/libraries/utils.py

```diff
@@ -46,31 +46,6 @@ def makedirs(path, mode=0o777, exists_ok=True):
     mounting._makedirs(path=path, mode=mode, exists_ok=exists_ok)
 
 
-def apply_yum_workaround(context=None):
-    """
-    Applies a workaround on the system to allow the upgrade to succeed for yum/dnf.
-    """
-    yum_script_path = api.get_tool_path('handleyumconfig')
-    if not yum_script_path:
-        raise StopActorExecutionError(
-            message='Failed to find mandatory script to apply',
-            details=reinstall_leapp_repository_hint()
-        )
-    cmd = ['/bin/bash', '-c', yum_script_path]
-
-    try:
-        context = context or mounting.NotIsolatedActions(base_dir='/')
-        context.call(cmd)
-    except OSError as e:
-        raise StopActorExecutionError(
-            message='Failed to exceute script to apply yum adjustment. Message: {}'.format(str(e))
-        )
-    except CalledProcessError as e:
-        raise StopActorExecutionError(
-            message='Failed to apply yum adjustment. Message: {}'.format(str(e))
-        )
-
-
```

Comment:
must be kept and marked for deprecation

Reply:
Maybe, didn't think so though - Because it doesn't make much sense to be used other than the current use cases.

Reply:
that's true, but keeping it and set it as deprecated does not hurt us. I prefer to keep our promise about the deprecation until we document / specify functions/models/.... are not supposed to be used by others.